rskj-core/src/main/java/co/rsk/net/BlockProcessResult.java

```diff
@@ -69,10 +71,17 @@ public boolean isInvalidBlock() {
 
     private void logResult(String blockHash, Duration processingTime) {
```

Comment:
I would add some tests for this method, seems to have some complexity.

Reply:
Done